velox/exec/fuzzer/MemoryArbitrationFuzzer.cpp

```diff
@@ -120,6 +135,31 @@ class MemoryArbitrationFuzzer {
     return boost::random::uniform_int_distribution<int32_t>(min, max)(rng_);
   }
 
+  // Scoped directory that, if enabled fault injection, makes sure the fault
+  // injection is cleared upon its destruction.
+  class ScopedFileDirectory {
```

Comment:
Do we need this abstraction?

Reply:
Yeah we can maybe remove it since it is currently only doing the cleanup.